docs/configuring-dns.md

```diff
@@ -40,6 +41,8 @@ As the table above illustrates, you need to create 2 subdomains (`matrix.<your-d
 The `riot.<your-domain>` subdomain is necessary, because this playbook installs the Riot web client for you.
 If you'd rather instruct the playbook not to install Riot (`matrix_riot_web_enabled: false` when [Configuring the playbook](configuring-playbook.md) later), feel free to skip the `riot.<your-domain>` DNS record.
 
+The `dimension.<your-domain>` subdomain is necessary, because this playbook installs the Dimension integrations manager for you. Dimension installation is disabled by default and if you explicitly do not enable it's installation (`matrix_dimension_enabled: true` when [Configuring the playbook](configuring-playbook.md) later), feel free to skip the `dimension.<your-domain>` DNS record.
```

Comment:
There's a typo (`it's` -> `its`).

I'm also wondering if we should use some better wording to stress on the fact that Dimension is optional.
For example, instead of "is necessary" we could say "may be necessary" (if you enable it..).

We could also link to the "Setting up Dimension" page, instead of saying this: "(`matrix_dimension_enabled: true` when [Configuring the playbook](configuring-playbook.md) later)", because setting up Dimension is more involved than just `matrix_dimension_enabled: true`. It does require an access token, etc.

Example of a reworded message:

```
The `dimension.<your-domain>` subdomain may necessary, because this playbook could install the [Dimension integrations manager](http://dimension.t2bot.io/) for you. Dimension installation is disabled by default, because it's only possible to install it after the other Matrix services are working (see [Setting up Dimension](docs/configuring-playbook-dimension.md) later). If you do not wish to set up Dimension, feel free to skip the `dimension.<your-domain>` DNS record.
```